installazione_archlinux_markdown.md

```diff
@@ -666,7 +670,18 @@ cd <nomeaurhelper>
 makepkg -si
 ```
 
-Installiamone quindi uno: **yay** ! Per lo step successivo è <u>**fortemente consigliato**</u> l'accesso con l'utente e <u>non con root</u>.
+Attualmente le principali alternative sono **yay** e **paru**, la prima scritta in Go e la seconda, più recente, in Rust.
+Installiamone quindi uno: 
+
+- **paru** ! Per lo step successivo è <u>**fortemente consigliato**</u> l'accesso con l'utente e <u>non con root</u>.
```

Comment:
no-sense duplicare la sezione dell 'aur helper quando **esattamente al comando prima** è stato specificato un modo **generico** per installarli, quello di yay è un esempio successivo che serve a concretizzare
lascia pure come consiglio quello di paru volendo, ma metti solo un installazione.  

## Modifica consigliata
lascia yay, e sopra scrivi *come una lista* solo i package manager consigliati 
esempio:  

Attualmente le alternative che mi sento di consigliare sono:  
- yay (scritta in go)
- paru ( scritta in rust) 

Ne si può trovare una tabella descrittiva [a questa pagina](https://wiki.archlinux.org/index.php/AUR_helpers)  
Installiamone quindi uno: **yay** ! Per lo step successivo è <u>**fortemente consigliato**</u> l'accesso con l'utente e <u>non con root</u>. Come specificato sopra, in questo caso basta solo sostituire la parola **yay** a quella **`<nome aurhelper>`**

```bash
git clone https://aur.archlinux.org/yay.git
cd yay
makepkg -si
```

Per installare **paru**, cambiare scriverlo quindi al posto di aur.   

## ALCUNE NOTE
Preferibilmente lascia yay a paru, se vuoi evitare di cambiare tutto il resto dei riferimenti

Reply:
Banalmente basterebbe mantenere `yay` e aggiungere una nota che spiega come installare ed usare `paru` al suo posto.

Ecco sarebbe ottimo anche trovare qualche beneficio, se ne ha.

Reply:
esiste già il comando che spiega come installare genericamente ogni aurhelper
```bash
git clone https://aur.archlinux.org/<nomeaurhelper>.git
cd <nomeaurhelper>
makepkg -si
```

Reply:
Non ho idea del perché abbia citato `paru` ma se porta qualche beneficio in più si potrebbe valutare una piccola nota (?)

In caso contrario resta solo uno dei tanti helper.

Reply:
Sulla tabella descrittiva di arch hanno gli stessi `yes`. by the way va benissimo citarlo in una lista di "consigliati", può anche lasciarlo nell'installazione volendo, ma l'altro va eliminato
specificare due installazioni appesantisce la lettura inutilmente imo. 
appena corregge ste due change request faccio il merge immediato

Reply:
"Forse" i pregi sono nel linguaggio usato? Ma lo vedo più come un punto tecnico che di interesse per l'utente finale.

Reply:
si può scriverlo volendo, una volta lo feci con `pakku` spiegando i vantaggi del linguaggio `NIM`

Reply:
https://www.reddit.com/r/archlinux/comments/jjn1c1/paru_v100_and_stepping_away_from_yay/?utm_medium=android_app&utm_source=share